### PR TITLE
fix: bug with maximum and minimum tickets per order

### LINKED ISSUE #5235: error when the number of minimum tickets is less than maximum. No error when maximum is less than minimum.
**Describe the bug**
On creating a new event, when the number of tickets per order is less than maximum tickets, an error is shown, the error being "Minimum order should not be greater than maximum". Also, when we increase the number of minimum tickets, we can somehow end up having more minimum tickets than maximum tickets with no error.

**To Reproduce**
Steps to reproduce the behaviour:
1.  Go to https://eventyay.com/create.
2. Click on free ticket in the ticketing system.
3. increase minimum ticket by 1 (ie : 2) and reduce maximum ticket by 1 (ie: 9).
4. See error(1)
5. Now keep increasing the minimum ticket number till it becomes more than the maximum ticket number.
6. there will be no error(2).

**Expected behaviour**
1. In the first case there should be no error since the minimum ticket number is less than the maximum ticket number.
2. In the second case the error should be prompted since the minimum ticket number is greater than maximum ticket number. 

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

### PR DESCRIPTION
**Pull request by Soumi150**
Fixes #5235

**Short description of what this resolves:**
When user changes minimum order to greater than maximum or vice versa ,prompt will be given.

**Checklist**
- [x] I have read the Contribution & Best practices Guide.
- [x] My branch is up-to-date with the Upstream development branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/wizard/basic-details-step.js [hunk 1]
```diff
@@ -139,13 +139,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
...
     $.fn.form.settings.rules.checkMaxMinOrder = () => {
-      return $('.ui.form').form('get value', 'ticket_min_order') <= $('.ui.form').form('get value', 'ticket_max_order');
+      return parseInt($('.ui.form').form('get value', 'ticket_min_order'),10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'),10);
     };
 
     const validationRules = {
       inline : true,
       delay  : false,
       on     : 'blur',
+      on     :'change',
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:149]: Codacy found an issue: [Missing space before value for key 'on'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6290947)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6290947&bid=20466595)

File: app/components/forms/wizard/basic-details-step.js [hunk 2]
```diff
@@ -139,13 +139,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
       return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
     };
     $.fn.form.settings.rules.checkMaxMinOrder = () => {
-      return $('.ui.form').form('get value', 'ticket_min_order') <= $('.ui.form').form('get value', 'ticket_max_order');
+      return parseInt($('.ui.form').form('get value', 'ticket_min_order'),10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'),10);
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:142]: Codacy found an issue: [A space is required after ','.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6290947)

File: app/components/forms/wizard/basic-details-step.js (hunk 1 above)
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:149]: Codacy found an issue: [Duplicate key 'on'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6290947)

**Comment by iamareebjamal** [comment]
@maze-runnar Can you please review this and try it out

**Comment by maze-runnar** [comment]
working fine

**Review by maze-runnar — APPROVED** [review]

**Pull request merged**